### PR TITLE
Added padding to the create forum button

### PR DESCRIPTION
**Pull request by anubis2814**
The link click area was too large. Added top padding that made it better.

### CODE REVIEW

File: view/theme/frio/style.php [hunk 1]
```diff
@@ -208,8 +208,8 @@
 	'$background_repeat'           => $background_repeat,
 	'$login_bg_image'              => $login_bg_image,
 	'$login_bg_color'              => $login_bg_color,
-	'$font_color_darker'           => $font_color_darker ?? '#222',
-	'$font_color'                  => $font_color ?? '#444',
+	'$font_color_darker'           => $font_color_darker ?? '#000',
+	'$font_color'                  => $font_color ?? '#111',
```
Comment (MrPetovan) [inline view/theme/frio/style.php]: Please separate these changes in another PR to allow for discussion.

File: view/theme/frio/css/style.css [hunk 2]
```diff
@@ -76,6 +76,12 @@ blockquote {
 .fakelink {
 	cursor: pointer;
 }
+.createforum {
+	padding: 25px;
+	cursor: pointer;
+	padding-top: 5px;
+}
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: If you want to give a button appearance to the forum creation button, you can use the `btn btn-primary` classes instead of custom CSS.

File: src/Content/ForumManager.php [hunk 3]
```diff
@@ -154,7 +154,9 @@ public static function widget(string $baseurl, int $uid, int $cid = 0)
 					'$total'	=> $total,
 					'$visible_forums' => $visible_forums,
 					'$showless'	=> DI::l10n()->t('show less'),
-					'$showmore'	=> DI::l10n()->t('show more')]
+					'$showmore'	=> DI::l10n()->t('show more'),
+					'$create_forum'	=> DI::l10n()->t('+ Create a new Forum?')
+					]
```
Comment (MrPetovan) [inline src/Content/ForumManager.php]: I would remove the question mark, this is a button where the intent should be clear. For the plus sign, please use an icon from the forkawesome set like this: `<i class="fa fa-plus"></i>`. 
```suggestion
					'$create_forum'	=> DI::l10n()->t('Create a new Forum'),
				]
```

File: view/templates/widget_forumlist.tpl [hunk 4]
```diff
@@ -45,6 +46,11 @@ function showHideForumlist() {
 		{{if $total > $visible_forums }}
 		<li onclick="showHideForumlist(); return false;" id="forum-widget-collapse" class="forum-widget-link fakelink tool">{{$showmore}}</li>
 		{{/if}}
+
+		<a href='register' class= "createforum">{{$create_forum}} </a>
+		
+
+
```
Comment (MrPetovan) [inline view/templates/widget_forumlist.tpl]:
```suggestion
		<li>
			<a href="register" class="btn btn-primary"><i class="fa fa-plus"></i> {{$create_forum}}</a>
		</li>
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: view/theme/frio/style.php (hunk 1 above)
Comment (anubis2814) [reply view/theme/frio/style.php]: Crap i thought I had unchanged that since someone else was already working on it.

Comment (anubis2814) [reply view/theme/frio/style.php]: Is there are way to just straight up reject this change? It was an accident.

Comment (MrPetovan) [reply view/theme/frio/style.php]: Sure, you can do an interactive rebase onto `develop` and drop the c78e2345a09fc99660c2564778442d2929ccbf9d commit from the branch.

File: view/theme/frio/css/style.css (hunk 2 above)
Comment (anubis2814) [reply view/theme/frio/css/style.css]: Would you prefer a button over a link?

Comment (MrPetovan) [reply view/theme/frio/css/style.css]: By adding padding and changing the cursor look, you’ve effectively changed the link into a button, I’m just picking up what you put down.

If it’s meant to be a link, the CSS isn’t necessary.

Comment (anubis2814) [reply view/theme/frio/css/style.css]: Here is what it looks like on screen, but with a little more padding above.  Let me know if you like how it looks or it should be an actual button, and if I should use the FA plus sign on it.

Comment (MrPetovan) [reply view/theme/frio/css/style.css]: Thank you for the screenshot, now that I'm looking at it I feel like the button to create a new forum should be a plus icon in the same place where the "Edit" icon is in the "Groups" widget, at the top right.

Comment (anubis2814) [reply view/theme/frio/css/style.css]: Oh yeah that will look much more sleek and minimalistic.  I'll do that.
